### PR TITLE
Bring primary CI back up

### PR DESCRIPTION
**Pull request by edwardcwang**

### CODE REVIEW

File: ._travis.yml [hunk 1]
```diff
@@ -10,6 +10,7 @@ script:
   - source sourceme.sh
   - cd src/test
   - pip install mypy
-  - ./mypy_with_exclusions.sh
+# Travis' mypy install causes some errors
```
Comment (colinschmidt) [inline ._travis.yml]: For the record I believe I see these errors as well thats why I made the other changes. So its some version issue that would appear for others.

**Comment by edwardcwang** [comment]
Also tested with #374 and it seems to be responding now.

File: ._travis.yml (hunk 1 above)
Comment (edwardcwang) [reply ._travis.yml]: Ah good point, added the version number for mypy.

**Comment by edwardcwang** [comment]
Will plan to this tomorrow unless other major issues crop up

**Comment by edwardcwang** [comment]
Thanks!

**Comment by colinschmidt** [comment]
I think we should leave travis checks on until the buildbot code is available.

**Pull request merged**